src/CDN.elm

```diff
@@ -23,22 +25,36 @@ To use, just put the stylesheet you want in your top-level view function, like s
             , ...
             ]
 
+# CSS Frameworks
+
+These are frameworks that are more opinionated, meaning they tend to style things
+for you in a default way and provide more structure.
+
 @docs bootstrap
 @docs skeleton
-@docs reset
+@docs pure
 @docs foundation
-@docs fontAwesome
 @docs bulma
-@docs pure
-@docs bass
 @docs wing
+
+# CSS Libraries
+
+These are more "mix and match" css libaraies.
+
+@docs reset
+@docs fontAwesome
+@docs bass
+@docs colors
+@docs tachyons
 -}
 
 import Html exposing (..)
 import Html.Attributes exposing (..)
 
 
 {-| [Bootstrap 3](http://getbootstrap.com/)
+Bootstrap is the most popular HTML, CSS, and JS framework for developing
```

Comment:
I like the idea of adding the project's blurb to the docs, but since Elm docstrings support markdown, I think we should mark these as quotes with `>`.